packages/vscode/src/html/autoClose.ts

```diff
@@ -12,78 +12,85 @@ import { TextDocumentContentChangeEvent } from 'vscode-languageserver-protocol';
 
 /**  */
 export function activateTagClosing(
-  tagProvider: (document: TextDocument, position: Position) => Thenable<string>,
-  supportedLanguages: { [id: string]: boolean },
-  configName: string
+	tagProvider: (document: TextDocument, position: Position) => Thenable<string>,
```

Comment:
This file somehow escaped the formatting changes we did a long time ago. Not sure how